addon/utils/model-factory.js

```diff
@@ -33,16 +33,21 @@ function createPropertyTreeFromFields(
       sourceGraph,
       sourceNode,
     });
-    let groupUri = store.any(fieldUri, SHACL('group'), undefined, formGraph);
-    const group = groups[groupUri.value];
+    let sectionUri = sectionHelpers.containingItem(fieldUri, {
```

Comment:
nitpick: Not sure why but in my head `sectionHelpers` referes to a variable and `SectionHelpers` refers to a helper ? idk  😆 

(Maybe it comes from a previous project where helpers where classes and those where always starting with a capital letter)